Assets/Scripts/Strategy/GameResources/City.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using SwordAndBored.Strategy.TimeSystem.Subscribers;
+using SwordAndBored.Strategy.TimeSystem.TimeManager;
+
+namespace SwordAndBored.Strategy.GameResources
+{
+    public class City : MainThreadPostTimeStepSubscriber
+    {
+        public ResourceManager resourceManager { get; set; }
+        public bool underPlayerControl = true;
+
+        void Awake()
+        {
+            underPlayerControl = true;
+        }
+        protected override void MainThreadPostTimeStepUpdate()
+        {
+            if(underPlayerControl)
+            {
+                resourceManager.GoldAmount += 1;
+            }
+        }
```

Comment:
Should a city increase gold amount? Or should there be a GoldCity that extends this class that increases gold

Reply:
Fixed in commit 0c427fe